protocol.html

```diff
@@ -376,24 +375,27 @@ <h2 about="#introduction" property="schema:name" typeof="deo:Introduction">Intro
                 <h3 property="schema:name">Overview</h3>
                 <div datatype="rdf:HTML" property="schema:description">
                   <p><em>This section is non-normative.</em></p>
-
-                  <p>The following diagram shows the high-level interactions involved discovering notification channels and subscription resources to towards receiving notifications.</p>
+                  <p>The Solid Notification protocol defines a mechanism for client applications to discover notification services avaiable for any given resource.</p>
+                  <p>The following diagram shows the high-level interactions involved in discovering notification channels and subscription resources. A subscription resource allow Subscription Clients to request a customized notification channels.</p>
```

Comment:
```suggestion
                  <p>The following diagram shows the high-level interactions involved in discovering notification channels and subscription resources. A subscription resource allow Subscription Clients to request a customised notification channels.</p>
```

Reply:
Can you clarify if we are using British or American English? With British English your suggestion here is correct. However, W3C Style Guide expects American English. In which case we should use "customized".

Reply:
It is a practise, not a requirement. There are no "pubrules" (publication rule) that prohibits the use of different spellings - that would be very counter to the W3C Internationalization activities. Or any human checking or blocking publication because of it - as far as I know! Furthermore, the document's language is set to `en` without a subtag (which would allow even mixed spelling) - from the point of internationalisation-friendliness, it is more important that the document uses the `lang` attribute among other things, than not. See also https://validator.w3.org/i18n-checker/check?uri=https%3A%2F%2Fsolid.github.io%2Fnotifications%2Fprotocol - "No issues to report".

Having said that, editors should indeed make use of the guides available to them as much as possible and as they deem useful. This is also mentioned in Solid contributing guidelines. If no objections, I prefer to continue to use British or Canadian over American spelling, but most importantly consistently within a given document, which is why I reverted your request.

Reply:
Should we specify the lang as en-GB explicitly, then? The thing is, an American English inclined contributor would likely be inconsistent again in the future otherwise...